### DISCUSSION TITLE
Would using bootstraploader.xpi style All-in-one script make install process easier?

### DISCUSSION THREAD

**Discussion by 117649** [onemen/TabMixPlus#261] [General]
@onemen We may seeing a little increase on install problems.
And I have this idea for a while.
bootstraploader.xpi use all the same scripts files as stand alone scripts but they are all nested inside one big script.
What if we use this practice for userchrome install method?
For a bad example if we nested all needed scripts inside 'config.js' the install process would be as simple as place that file into FF install folder.

**Comment by onemen** [comment]
`config.js` control Firefox itself and all the other code is installed per profile.

I am planing on working on automatic installer for the scripts, but it will take few months.
